content/about/security-release-process/security-release-process.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Security Release Process
+description: Release process for Istio security vulnerabilities.
+weight: 35
+page_icon: /img/security-release.svg
+---
+
+The Istio team follows the process below when releasing patches for critical
+security vulnerabilities.
+
+## Security vulnerability disclosures
+
+Please refer to the [disclosure process](/about/security-vulnerabilities/).
+
+## Istio security team
+
+Security vulnerabilities should be handled quickly and privately.
+The primary goal of this process is to reduce the total time customers are
+vulnerable to publicly known exploits.
+
+The Istio security team is responsible for organizing the entire response
+including internal communication and external disclosure. The team enlists help
+from relevant developers and release leads to successfully run this process.
+
+The Istio security team consists of volunteers subscribed to the
+private [`istio-security-vulnerabilities@google.com`](mailto:istio-security-vulnerabilities@google.com)
+list.
+
+### Istio security team membership
+
+Membership to the Istio security team is based on the roles defined below.
+
+#### Roles
+
+* Fix lead
+
+Per issue, Fix Lead sees the issue through to the end.
+
+* Release lead
+
+Per issue, Release Lead is in charge of releasing the fix.
+
+* Disclosure lead
+
+Handles public messaging around the bug, and the documentation of how to upgrade.
+Explains the severity of the vulnerability and requests CVEs (Common Vulnerability Exposures).
+
+* Triage
+
+Makes sure the relevant people are notified, also responds to issues
+that are not actually security vulnerability issues.
+This person is the escalation path for a vulnerability if it is one.
+
+* Infra
+
+Makes sure the fixes are tested appropriately. This person is the build cop.
+It is the person you call when you need help testing, or the release branch is all
+messed up.
+
+#### Scheduled rotation
+
+Each week has a primary & secondary for each role. If something comes up that week,
+primary owns it. Secondary is there if we can't get a hold of the primary.
+
+## Patch, release, and public communication
+
+For each vulnerability, Fix Lead drives the schedule using his/her best
+judgment based on severity, development time, and the feedback from Release Lead.
+If the fix relies on another upstream project's disclosure timeline, that will
+adjust the process as well. Fix Lead will work with the upstream project
+owners to determine the fix timeline and best protect the users.
```

Comment:
To fix grammar and improve clarity, I suggest:

For each vulnerability, the **fix lead** drives the schedule using their best
judgment based on the severity, development time, and the feedback from the **release lead**.
If the fix relies on another upstream project's disclosure timeline, that dependency causes the process to adjust. The **fix lead** works with the upstream project owners to determine the fix timeline to best protect the users.